app/controllers/replies_controller.rb

```diff
@@ -97,4 +97,8 @@ def find_new_thread
       @new_thread = NewThread.friendly.find_by_id(params[:new_thread_id])
     end
   end
+
+  def friendly_url
+    @new_thread = NewThread.friendly.find(params[:new_thread_id])
+  end
 end
```

Comment:
For replies controller you need to set `@reply` in your before filter. This would fail when you write test.